featurebyte/session/session_helper.py

```diff
@@ -66,6 +68,56 @@ async def validate_output_row_index(session: BaseSession, output_table_name: str
         raise ValueError("Row index column is invalid in the output table")
 
 
+async def run_coroutines(coroutines: List[Coroutine[Any, Any, None]]) -> None:
+    """
+    Execute the provided list of coroutines
+
+    Parameters
+    ----------
+    coroutines: List[Coroutine[Any, Any, None]]
+        List of coroutines to be executed
+
+    Raises
+    ------
+    Exception
+        If any task failed
+    """
+    tasks = [asyncio.create_task(coro) for coro in coroutines]
+    try:
+        await asyncio.gather(*tasks)
```

Comment:
Would it be better if we return `asyncio.gather(*tasks)`? Understood that the output is not used at the moment.